tokio/src/sync/mpsc/bounded.rs

```diff
@@ -314,7 +314,8 @@ impl<T> Sender<T> {
         Sender { chan }
     }
 
-    /// Sends a value, waiting until there is capacity.
+    /// Sends a value, waiting until there is capacity. In case you need a
+    /// polling interface, use [`PollSender`].
```

Comment:
I would like the link to be in the same place in the docs as where I put the similar thing for `Semaphore`.